docker/oso-image-inspector/centos7/root/config.yml

```diff
@@ -17,10 +17,10 @@
 #      hour: "13"
 #      job: "/usr/local/bin/cron-in-memory-scan.sh &>/dev/null"
 
-    - name: upload_scanlogs
-      minute: "15"
-      hour: "14"
-      job: "/usr/local/bin/upload_scanlogs &>/dev/null"
+#    - name: upload_scanlogs
+#      minute: "15"
+#      hour: "14"
+#      job: "/usr/local/bin/upload_scanlogs &>/dev/null"
```

Comment:
Why commented out?  Would prefer upload_scanlogs.sh if it's a shell script to make that more clear.